testsuite/integration-arquillian/tests/base/pom.xml

```diff
@@ -372,6 +372,12 @@
             <plugin>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>${maven-compiler-plugin.version}</version>
+                <configuration>
+                    <!-- #14072 Execute AddUserTest only with auth-server-wildfly -->
+                    <testExcludes>
+                        <testExclude>org/keycloak/testsuite/adduser/**</testExclude>
```

Comment:
We have some annotations that let you exclude some tests for the Quarkus dist. Did you look at that approach?

Reply:
Yes, I'm aware of that. However, AFAIK, it's not suitable for this situation. 

We need to exclude the test in the compile time and not within the runtime. To be more specific, the test uses `org.keycloak.wildfly.adduser.AddUser` class, which is included in the `keycloak-wildfly-adduser` artifact, but the artifact is present only when those tests should be executed with WF.

I was thinking about removing the artifact and the test completely from the testsuite, but that test should be present in the testsuite. We should include those test cases in the "Quarkus" testsuite as we don't have any similar check for that(AFAIK). I'd like to create some follow-up task for that.